pkg/network/setup/netstat_test.go

```diff
@@ -302,6 +302,40 @@ var _ = Describe("netstat", func() {
 			Expect(setup.NetStat.PodInterfaceVolatileDataIsCached(setup.Vmi, primaryNetworkName)).To(BeFalse())
 			Expect(setup.NetStat.PodInterfaceVolatileDataIsCached(setup.Vmi, secondaryNetworkName)).To(BeFalse())
 		})
+
+		It("run status and expect 1 attached iface & 1 detached iface to be reported based on multus status and guest-agent data", func() {
```

Comment:
I think this is not covering all the paths of the new code.

Looking at the logic, we need also: Appears in the original status as multus, but it is no longer in the spec.

Reply:
Added test for the suggested scenario